lib/transaction/index.js

```diff
@@ -747,19 +746,13 @@ function _linkExceptionToSegment(exception) {
     return
   }
 
-  const config = this.agent.config
-
-  // Add error attributes to the span
-  const details = exception.getErrorDetails(config)
-  segment.addSpanAttribute('error.message', details.message)
-  segment.addSpanAttribute('error.class', details.type)
-
-  const isExpected =
-    errorHelper.isExpectedErrorClass(config, details.type) ||
-    errorHelper.isExpectedErrorMessage(config, details.type, details.message)
-
-  if (isExpected) {
-    segment.addSpanAttribute('error.expected', isExpected)
```

Comment:
This work got pushed to the span event creation. Now we are just setting the details on the span context. If the transaction has an errored status code, these details will not be used when the span is created.

Reply:
Did you mean ignored here instead of errored?

Reply:
yeah... i meant "ignored error status code"